Rakefile

```diff
@@ -0,0 +1,25 @@
+require 'json'
+require 'open-uri'
+require 'html/proofer'
+
+namespace :test do
+  task :travis do
+    if (pr = ENV['TRAVIS_PULL_REQUEST']) && !pr.empty?
+      sh 'git clone https://github.com/stbenjam/katello.org.git -b customize-path deploy'
```

Comment:
Will change to this when https://github.com/Katello/katello.org/pull/121 is merged:

```
sh 'git clone https://github.com/Katello/katello.org.git -b deploy deploy'
```